physics/dynamic_frame.hpp

```diff
@@ -35,14 +36,14 @@ class DynamicFrame {
   virtual RigidMotion<InertialFrame, ThisFrame> ToThisFrameAtTime(
       Instant const& t) const = 0;
   virtual RigidMotion<ThisFrame, InertialFrame> FromThisFrameAtTime(
-      Instant const& t) const = 0;
+      Instant const& t) const;
```

Comment:
The asymmetry makes me sad.  Could we leave it to the subclass which one they compute primarily and which one is the inverse?